runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/Swift.test.stg

```diff
@@ -96,7 +102,7 @@ ParserPropertyCall(parser, property) ::= <<
 <parser>.<property>
 >>
 
-PositionAdjustingLexer() ::= <<
```

Comment:
Why is this gone in Swift but not other targets?

Reply:
it certainly escaped me